src/ApiCompat/project.json

```diff
@@ -1,6 +1,6 @@
 {
   "dependencies": {
-    "Microsoft.Cci": "4.0.0-rc3-24214-00",
+    "Microsoft.Cci": "4.0.0-rc4-24217-00",
```

Comment:
It's because `ProjectKTfsCurrentRef` stayed what it was when we RTM'd, but `DependencyBranch` turned to master--so we got what we last built on "master" (ProjectK) when we RTM'd. https://github.com/dotnet/versions/blob/b6fe7bf0d879e7a750b15e90743d5191e3b478de/build-info/dotnet/projectk-tfs/master/Latest_Packages.txt

We should either separate out the build-info branches, or update the CoreCLR and ProjectK SHAs to latest too. For the most part this should have no effect because dependencies are mostly stable at the moment.